app/javascript/lib/validation/validate-file-content.js

```diff
@@ -425,6 +425,9 @@ async function parseFile(file, fileType, fileOptions={}, sizeProps={}) {
     // get any unhandled or deliberate short-circuits
     if (error instanceof ParseException) {
       parseResult.issues.push(['error', error.key, error.message])
+    } else if (error instanceof TypeError) {
+      const msg = 'File cannot be uploaded in it\'s current state. Please reach out for assistance'
```

Comment:
Fixing trivial typo and adding period:

```suggestion
      const msg = 'File cannot be uploaded in its current state.  Please reach out for assistance.'
```